Super Tafani Member/index.html

```diff
@@ -0,0 +1,220 @@
+                        </svg>
+                        
+                        <p class="coral">Show details</p>
+                    </td>
+                  </tr>
+                  <tr>
+                    <td scope="row">999961</td>
+                    <td >Reply</td>
+                    <td>ibrahim_m123@stc.sa</td>
+                    <td>27 May 2022</td>
+                    <td>15 Reports</td>
+                    <td class="d-flex gap-2">
+                      <svg class="mt-1" width="20" height="16" viewBox="0 0 20 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M20 6.94C19.9896 6.84813 19.9695 6.75763 19.94 6.67V6.58C19.8919 6.47718 19.8278 6.38267 19.75 6.3L13.75 0.3C13.6673 0.222216 13.5728 0.158081 13.47 0.11H13.37C13.2728 0.0580033 13.1683 0.020922 13.06 0H3C2.20435 0 1.44129 0.316071 0.87868 0.87868C0.316071 1.44129 0 2.20435 0 3V13C0 13.7956 0.316071 14.5587 0.87868 15.1213C1.44129 15.6839 2.20435 16 3 16H17C17.7956 16 18.5587 15.6839 19.1213 15.1213C19.6839 14.5587 20 13.7956 20 13V7C20 7 20 7 20 6.94ZM14 3.41L16.59 6H15C14.7348 6 14.4804 5.89464 14.2929 5.70711C14.1054 5.51957 14 5.26522 14 5V3.41ZM18 13C18 13.2652 17.8946 13.5196 17.7071 13.7071C17.5196 13.8946 17.2652 14 17 14H3C2.73478 14 2.48043 13.8946 2.29289 13.7071C2.10536 13.5196 2 13.2652 2 13V3C2 2.73478 2.10536 2.48043 2.29289 2.29289C2.48043 2.10536 2.73478 2 3 2H12V5C12 5.79565 12.3161 6.55871 12.8787 7.12132C13.4413 7.68393 14.2044 8 15 8H18V13ZM5 8H9C9.26522 8 9.51957 7.89464 9.70711 7.70711C9.89464 7.51957 10 7.26522 10 7C10 6.73478 9.89464 6.48043 9.70711 6.29289C9.51957 6.10536 9.26522 6 9 6H5C4.73478 6 4.48043 6.10536 4.29289 6.29289C4.10536 6.48043 4 6.73478 4 7C4 7.26522 4.10536 7.51957 4.29289 7.70711C4.48043 7.89464 4.73478 8 5 8ZM5 10C4.73478 10 4.48043 10.1054 4.29289 10.2929C4.10536 10.4804 4 10.7348 4 11C4 11.2652 4.10536 11.5196 4.29289 11.7071C4.48043 11.8946 4.73478 12 5 12H15C15.2652 12 15.5196 11.8946 15.7071 11.7071C15.8946 11.5196 16 11.2652 16 11C16 10.7348 15.8946 10.4804 15.7071 10.2929C15.5196 10.1054 15.2652 10 15 10H5Z" fill="#FF375E"/>
+                        </svg>
+                        
+                        <p class="coral">Show details</p>
+                    </td>
```

Comment:
Please remove this `</td>` as it is a syntax error.

Reply:
done